site/source/docs/porting/Audio.rst

```diff
@@ -0,0 +1,143 @@
+.. _Audio:
+
+=====
+Audio
+=====
+
+Emscripten supports OpenAL 1.1 "out of the box", with no additional compilation flag required. The implementation uses the Web Audio API.
+
+You can reasonably expect ported OpenAL applications to "just work" with no additional effort.  
+However, there are some implementation-specific aspects which are worth considering - these are documented here.
+
+
+.. _Audio-openal-extensions-g:
+
+Supported OpenAL extensions
+===========================
+
+The following extensions are supported by Emscripten's OpenAL implementation.
+
+- ``ALC_SOFT_pause_device``;
+- ``ALC_SOFT_HRTF``;
+- ``AL_EXT_float32``;
+- ``AL_SOFT_loop_points``;
+- ``AL_SOFT_source_length``;
+- ``AL_EXT_source_distance_model``;
+- ``AL_SOFT_source_spatialize``;
+
+.. warning:: This doesn't mean you should just assume their presence ! For correctness, you should **always** check if an extension is supported before using it, like a good application would do.
+
+
+.. _Audio-guidelines-g:
+
+Guidelines for Audio on Emscripten
+==================================
+
+First: Do not loop on OpenAL calls.
```

Comment:
This would be good to be clarified to refer to synchronously blocking on long running calls to be more clear. (accessing OpenAL calls in a loop is fine, one just can't block for long periods)